apis/spark/src/main/scala/io/tiledb/vcf/TileDBVCFHailReader.scala

```diff
@@ -0,0 +1,53 @@
+package io.tiledb.vcf
+
+import java.net.URI
+
+import is.hail.expr.SparkAnnotationImpex
+import is.hail.expr.ir.{ExecuteContext, MatrixHybridReader, TableRead, TableValue}
+import is.hail.types.physical.{PStruct, PType}
+import is.hail.types.virtual.{TArray, TLocus, TString, TStruct}
+import is.hail.types.{MatrixType, TableType}
+import is.hail.variant.{Locus, ReferenceGenome}
+import org.apache.spark.sql.{DataFrame, Row}
+
+import scala.collection.mutable
+
+
+class TileDBHailVCFReader(val uri: URI, val options: VCFDataSourceOptions, val df: DataFrame) extends MatrixHybridReader {
+  val reader: VCFDataSourceReader = new VCFDataSourceReader(uri, options)
```

Comment:
Is the options and reader needed? Isn't the dataframe passed in complete? So what is this for?